scvi/inference/trainer.py

```diff
@@ -94,20 +91,18 @@ def compute_metrics(self):
         if self.frequency and (epoch == 0 or epoch == self.n_epochs or (epoch % self.frequency == 0)):
             with torch.set_grad_enabled(False):
                 self.model.eval()
-                if self.verbose:
-                    logging.info("\nEPOCH [%d/%d]: " % (epoch, self.n_epochs))
+                logging.debug("\nEPOCH [%d/%d]: " % (epoch, self.n_epochs))
 
                 for name, posterior in self._posteriors.items():
                     logging.info_name = ' '.join([s.capitalize() for s in name.split('_')[-2:]])
                     if hasattr(posterior, 'to_monitor'):
                         for metric in posterior.to_monitor:
                             if metric not in self.metrics_to_monitor:
-                                if self.verbose:
-                                    logging.info(logging.info_name, end=' : ')
-                                result = getattr(posterior, metric)(verbose=self.verbose)
+                                logging.debug(logging.info_name, end=' : ')
```

Comment:
@gabmis logging.info_name should be replaced also by the logger ?

Reply:
ok, I thought this was a useful property of the logger

Reply:
I have no idea